test/unit/sitemap/property_boost_calculator_test.rb

```diff
@@ -2,129 +2,117 @@
 require "sitemap/sitemap"
 
 class PropertyBoostCalculatorTest < Minitest::Test
-  def test_documents_are_boosted_relative_to_default_for_property
+  def test_boosts_are_between_0_and_1
```

Comment:
I would love to use property-based tests here because the values in these unit tests seem so over-specific. But as far as I know there's no good PBT library for Ruby.